docs/content/en/docs/Getting started/setup-development/index.md

```diff
@@ -1,90 +1,172 @@
+The login console will look like "messed up" due to the logs of the installation process in the steps above. 
 
-## Using it
+{{< imgproc 1.png Fit "1280x500" >}}Immediatly after start. Nothing is messed up yet.{{< /imgproc >}}
+{{< imgproc 2.png Fit "1280x500" >}}"Messed up" login screen{{< /imgproc >}}
 
-You can now access Grafana and nodered via HTTP / HTTPS (depending on your setup). Default user for Grafana is admin. You can find the password in the secret RELEASE-NAME-grafana. Grafana is available via port 8080, nodered via 1880. 
+You can "clean it up" by pressing two times enter. 
 
+You can also immediatly proceed with entering the default username `rancher` (do not forget to press enter) and the default password `rancher` to login. 
+{{< imgproc 3.png Fit "1280x500" >}}Logging into k3OS using the username `rancher`{{< /imgproc >}}
+{{< imgproc 4.png Fit "1280x500" >}}Logging into k3OS using the password `rancher`{{< /imgproc >}}
+
+After a successfull login you should see the current IP address of the device on your computer screen.
+
+{{< imgproc 5.png Fit "1280x500" >}}Successfully logged into k3OS{{< /imgproc >}}
+
+
+#### Step 2: Enable SSH password authentification 
+
+Enable SSH password authentification in k3OS [^ssh-password-authentication]. This is currently not necessary anymore as the automated setup script will do that automatically, **therefore this step can be skipped**. This paragraph only exists to remind you that this setting is not the default behavior of k3OS and should be deactivated in production environments.
+
+For production environments we recommend using a certificate to authenticate, which is enabled by default. This can be archieved by modifying the cloud-init file and [linking to a public key stored on your GitHub account.](https://gist.github.com/dmangiarelli/1a0ae107aaa5c478c51e#ssh-setup-with-putty)
+
+[^ssh-password-authentication]: See also out guide: [Enabling k3os password authentication](/docs/tutorials/add-username-password-authentification-k3os-ssh/)
+
+#### Step 3: Connect via SSH
+
+Connect via SSH [^SSH-client] from your laptop with the edge device. The IP address is shown on the computer screen on your edge device (see also step 1). If it is not available anymore, you can view the current IP address using `ip addr`. 
+
+#### Step 4: Getting Kubernetes credentials
+
+Execute `cat /etc/rancher/k3s/k3s.yaml` in your SSH session on your laptop  to retrieve the Kubernetes credentials. Copy the content of the result into your clipboard.
+
+{{< imgproc k3s_secret_1.png Fit "1280x500" >}}Execute `cat /etc/rancher/k3s/k3s.yaml`{{< /imgproc >}}
+
+{{< imgproc k3s_secret_2.png Fit "1280x500" >}}Copy the content{{< /imgproc >}}
+
+Connect with the edge device and the Kubernetes credentials in your clipboard.
```

Comment:
Not easily understandable. Instead maybe something like:
"Connect the program Lens with the edge device using the Kubernetes credentials from your clipboard."